pennylane/ops/qubit/qchem_ops.py

```diff
@@ -886,13 +886,14 @@ class OrbitalRotation(Operation):
 
     with the same orbital operation applied in the :math:`\alpha` and :math:`\beta` spin orbitals.
 
-    .. figure:: ../../_static/qchem/orbital_rotation_decomposition_extended.png
+    .. figure:: ../../_static/qchem/orbital_rotation.jpeg
         :align: center
         :width: 100%
         :target: javascript:void(0);
 
-    Here, :math:`G(\phi)` represents a single-excitation Givens rotation, implemented in PennyLane
-    as the :class:`~.SingleExcitation` operation.
+    Here, :math:`G(\phi)` represents a single-excitation Givens rotation and :math:`f\text{SWAP}(\pi)`
```

Comment:
We can probably add 1-2 sentences that this implementation is a modified version of the original paper and cite the paper.